stlab/concurrency/future.hpp

```diff
@@ -929,7 +934,7 @@ class future<T, enable_if_not_copyable<T>> {
     }
 
     template <typename S, typename F>
-    auto recover(S&& s, F&& f) && {
+    auto recover(S s, F&& f) && {
```

Comment:
Why not keep the forwarding reference on S?

Reply:
Fixed.